lib/elixir/test/elixir/task/supervisor_test.exs

```diff
@@ -22,6 +22,16 @@ defmodule Task.SupervisorTest do
     number
   end
 
+  def sleep_and_return_ancestor(number, _another_arg \\ nil) do
```

Comment:
Should we rather implement this as:

```elixir
defp sleep_and_return_ancestors(number, :another_arg) do
  sleep_and_return_ancestors(number)
end

defp sleep_and_return_ancestors(number) do
  # code today
end
```

So we assert the argument is correct?